python/dlisio/ext/core.cpp

```diff
@@ -835,6 +835,21 @@ PYBIND11_MODULE(core, m) {
         })
     ;
 
+    m.def( "parse_set_types", [](const std::vector< dl::record >& recs ) {
```

Comment:
Docs might be useful as it's not clear that it would return
'FILE-HEADER', 'CHANNEL', 'CHANNEL', 'encrypted', 'FRAME' 
sequential, but with omitted iflrs.